criu/crtools.c

```diff
@@ -422,6 +422,9 @@ int main(int argc, char *argv[], char *envp[])
 "			Namespace can be specified as either pid or file path.\n"
 "			OPTIONS can be used to specify parameters for userns:\n"
 "			    user:PID,UID,GID\n"
+"* Sanity checks:\n"
+"  --check-mounts        Check that mount trees restored fine in each mntns,\n"
+"                        check they are identical to what we have in images.\n"
```

Comment:
Perhaps this should be added to the man page (`Documentation/criu.txt`) too?

Reply:
done.